scripts/release/preparepackages.js

```diff
@@ -193,6 +194,25 @@ const tasks = new Listr( [
 							concurrency: cliArguments.concurrency
 						} );
 					}
+				},
+				{
+					title: 'Preparing CDN files.',
+					task: async () => {
+						await fs.copy( './dist/browser', `./${ RELEASE_CDN_DIRECTORY }/` );
+						await fs.copy( './dist/translations', `./${ RELEASE_CDN_DIRECTORY }/translations/` );
+						await fs.copy( './dist/browser', `./${ RELEASE_ZIP_DIRECTORY }/ckeditor5/` );
+						await fs.copy( './dist/translations', `./${ RELEASE_ZIP_DIRECTORY }/ckeditor5/translations/` );
+						await fs.copy( './scripts/assets/zip', `./${ RELEASE_ZIP_DIRECTORY }/` );
+
+						await fs.ensureDir( `./${ RELEASE_CDN_DIRECTORY }/zip` );
+						await tools.shExec(
+							`cd ${ RELEASE_ZIP_DIRECTORY } \
+							&& zip -r ../${ RELEASE_CDN_DIRECTORY }/zip/ckeditor5-${ latestVersion }.zip ./*`,
+							{ verbosity: 'error' }
+						);
+
+						await fs.copy( './build', `./${ RELEASE_CDN_DIRECTORY }/dll/ckeditor5-dll/` );
```

Comment:
This copies too many files:

![image](https://github.com/ckeditor/ckeditor5/assets/2270764/38d6496d-6149-4447-b79a-f536fe90867e)

Reply:
```
await fs.copy( `${ RELEASE_NPM_DIRECTORY }/ckeditor5/build`, `./${ RELEASE_CDN_DIRECTORY }/dll/ckeditor5-dll/` );
```

Let's copy it from the npm variant, as it's already filtered out.